.github/configuration/upgrade-test-configuration.yml

```diff
@@ -103,4 +103,14 @@ upgrade-version: [[
     version: target.latest,
     upgrade-type: minor
   }
+],
+[
```

Comment:
This seems unnecessary, let's update the above one from 15.3 -> latest to 15.4 ->latest

Reply:
Updated